src/components/input/clear/index.tsx

```diff
@@ -30,11 +30,10 @@ export class SmoothlyInputClear {
 				if (Editable.Element.is(parent)) {
 					parent.listen("changed", async p => {
 						if (Input.is(p)) {
-							this.disabled = p.readonly ? true : p.value ? !p.value : !p.changed
```

Comment:
What is the reason for remove this line?

Reply:
For clear/reset inside of inputs, we either wanna show or hide the clear/reset button - but never show and disable it.

But in the If statement below `(p instanceof SmoothlyForm)` there we sometimes wanna still show the clear/reset buttons but disable it.


You can see that this code mirrors smoothly-input-reset here:
https://github.com/utily/smoothly/blob/master-1.0/src/components/input/reset/index.tsx#L33